Client/src/Overview.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const OverView = styled.section`
+margin: 0;
+padding-top: 20px;
+border-top: 1px solid #ebebeb;
+display: block;
+`;
+
+const Row = styled.div`
```

Comment:
This doc has better descriptor names for React styled components. It's clear what each of them are styling within your rendered html